action.yml

```
@@ -1,21 +1,31 @@
-on:
-  workflow_call:  
+# The name of this file should be `action.yml` , `action.yaml` or `Dockerfile`.
 
-jobs:
-  formatting-check:
-    name: Formatting Check
-    runs-on: ubuntu-latest
-    steps:
+name: 'Check Code Style'
+desciption: 'Check files for correct code style'
+
+runs:
+  using: "composite"
+  steps:
     - uses: actions/checkout@v2
+
     - name: Run bootstrap
       run: |
         git submodule init
         git submodule update
         cd dev-tools/ && ./bootstrap
```

Let's leave this up to the caller too!

```suggestion
      run: cd dev-tools/ && ./bootstrap
```